### ISSUE TITLE
print mask changes  to log channel

### ISSUE THREAD

**Issue by jesopo** [Libera-Chat/maskwatch-bot#11]
this ends up looking confusing to onlookers:

```
<+bismite> [PV] <jess!~jess@libera/staff/jess> listmask
<+bismite> [PV] <jess!~jess@libera/staff/jess> setmask 2 lethal
```

what would be better is
```
<+bismite> [PV] <jess!~jess@libera/staff/jess> listmask
<+bismite> [PV] <jess!~jess@libera/staff/jess> setmask 2 lethal
<+bismite> SETMASK: type LETHAL /^jess-test!/i
```

or similar

**Comment by examknow** [comment]
should we do this for other commands (ie `addmask`, `togglemask`) as well or just setmask?

**Comment by jesopo** [comment]
`togglemask` definitely, `addmask` would be a bit redundant

**Comment by jesopo** [comment]
closed by #12
